core-library/src/main/java/org/silverpeas/core/socialnetwork/relationship/RelationShipService.java

```diff
@@ -48,6 +48,8 @@ public static RelationShipService get() {
 
   @Inject
   private RelationShipDao relationShipDao;
+  @Inject
+  private RelationShipEventNotifier relationShipEventNotifier;
 
   protected RelationShipService() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this constructor to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1213)